src/styles/index.css

```diff
@@ -0,0 +1,7 @@
+@import './index';
```

Comment:
The `@import` rule is used here, which is fine if you're using a preprocessor like SCSS. Ensure that this import is correctly handled during the build process, as plain CSS does not support importing SCSS files.